src/js/components/Graphs/Legend/legendHelper.jsx

```diff
@@ -0,0 +1,114 @@
+
+import React from 'react';
+
+import {
+  LegendHeader,
+  LegendItemIcon,
+  SquareIcon,
+  CircleIcon,
+  RectangleIcon,
+  LineIcon,
+  LegendItemDescription,
+  LegendItem,
+} from './legend.style';
+
+export const legendArrays = {
+  nodes: [
+    {
+      desc: 'missing',
+      color: 'yellow',
+      shape: 'square',
+    },
+    {
+      desc: 'unprocessed',
+      color: 'grey',
+      shape: 'square',
+    },
+    {
+      desc: 'success',
+      color: 'green',
+      shape: 'square',
+    },
+    {
+      desc: 'error',
+      color: 'red',
+      shape: 'square',
+    },
+    {
+      desc: 'other',
+      color: 'blue',
+      shape: 'square',
+    },
+  ],
+  composites: [
+    {
+      desc: 'startNode',
+      color: 'white',
+      shape: 'circle',
+    },
+    {
+      desc: 'endNode',
+      color: 'multi',
+      shape: 'circle',
+    },
+  ],
+  labels: [
+    {
+      desc: '_success',
+      color: 'green',
+      shape: 'rectangle',
+    },
+    {
+      desc: '_error',
+      color: 'red',
+      shape: 'rectangle',
+    },
+    {
+      desc: '_[custom]',
+      color: 'blue',
+      shape: 'rectangle',
+    },
+  ],
+  edges: [
+    {
+      desc: 'processed',
+      color: 'black',
+      shape: 'solid',
+    },
+    {
+      desc: 'unprocessed',
+      color: 'grey',
+      shape: 'dashed',
+    },
+  ],
+};
+
+export const createLegend = (title, items) => {
+  /* eslint-disable react/display-name */
+  const getIcon = {
```

Comment:
This should be outside of this function scope as this is static, no need to create that object every time function is called